wp-content/civi-extensions/goonjcustom/Civi/CollectionCampService.php

```diff
@@ -89,6 +90,37 @@ public static function getSubscribedEvents() {
     ];
   }
 
+  /**
+   *
+   */
+  public static function assignVolunteerAsCampInitiator(string $op, string $objectName, $objectId, &$objectRef) {
+    if ($op !== 'edit' || $objectName !== 'AfformSubmission') {
+      return FALSE;
+    }
+
+    try {
+      $data = $objectRef['data'] ?? [];
+      if (!$data) {
+        return;
+      }
+
+      $campId = $data['Eck_Collection_Camp1'][0]['id'];
+      $volunteerId = $data['Individual6'][0]['id'];
```

Comment:
What will happen in the future if we remove one individual and only five individual forms remain? Will this data fetching break, or is there another way to fetch this data?